make/run

```diff
@@ -57,28 +57,19 @@ kubectl get storageclass | grep --silent '(default)' 2>/dev/null || {
 }
 STORAGE_CLASS="${STORAGE_CLASS:=$(kubectl get storageclass 2>/dev/null | awk '/(default)/ { print $1 ; exit }')}"
 
-DOMAIN=${DOMAIN:-}
-NETWORK_ENV=bin/settings/network.env
-if test -n "${DOMAIN}"; then
-    TMP=$(mktemp -d)
-    cp "${NETWORK_ENV}" "${TMP}"
-    trap "rm -rf ${TMP}" EXIT
-    NETWORK_ENV="${TMP}/$(basename "${NETWORK_ENV}")"
-    sed -e "s/\\bcf-dev.io/${DOMAIN}/g" -i "${NETWORK_ENV}"
-fi
-source bin/settings/settings.env
-source "${NETWORK_ENV}"
+DOMAIN=${DOMAIN:-cf-dev.io}
+TCP_DOMAIN=tcp.${DOMAIN}
+UAA_HOST=uaa.${DOMAIN}
+INSECURE_DOCKER_REGISTRIES="\"insecure-registry.${DOMAIN}:20005\""
```

Comment:
This value is also included in `bin/kube_overrides.rb`; I wish it was specified only in one place.

Should it be included in `bin/settings.yaml`? I guess it can't work because we need to interpolate `DOMAIN`.

_sighs_

Reply:
will leave this alone